app/components-react/shared/inputs/FormFactory.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useEffect } from 'react';
+import { FormProps } from 'antd/lib/form';
+import debounce from 'lodash/debounce';
+import * as inputs from './inputList';
+import { TSlobsInputProps } from './inputs';
+import Form, { useForm } from './Form';
+import { TInputMetadata } from './metadata';
+
+type TInputValue = string | number | boolean;
+
+const componentTable: Dictionary<React.FunctionComponent<TSlobsInputProps<{}, TInputValue>>> = {
```

Comment:
For a slightly better type, you could specify that only `TInputType` is a valid key for the object.

```TypeScript
const componentTable: { [k in TInputType]: React.FunctionComponent<TSlobsInputProps<{}, TInputValue>> } = ...
```